src/main/resources/NebulaDaoBasic.xml

```diff
@@ -284,6 +284,13 @@
     <delete id="deleteById">
         DELETE VERTEX ${ ng.valueFmt( p0 ) }
     </delete>
+
+
+    <delete id="deleteByIdBatch">
+        @for ( v in ng_args[0] ) {
+        DELETE VERTEX ${ ng.valueFmt( v )};

```

Comment:
The use of passing in multiple ids at the same time may be more appropriate than the use of multiple statement structures.
https://docs.nebula-graph.com.cn/3.6.0/3.ngql-guide/12.vertex-statements/4.delete-vertex/
```cql
DELETE VERTEX <vid> [ , <vid> ... ] [WITH EDGE];
```

Reply:
OK, I learn that and will change it later!